plugins/velocity/src/main/java/org/apache/struts2/views/velocity/StrutsVelocityContext.java

```diff
@@ -96,11 +96,15 @@ protected Object chainedContextGet(String key) {
             return null;
         }
         for (VelocityContext chainedContext : chainedContexts) {
-            Object val = chainedContext.internalGet(key);
+            Object val = chainedContext.get(key);
```

Comment:
Was technically a bug as we were failing to look at chained contexts within our chained contexts